### ISSUE TITLE
About fine-tuning VGGT Omega

### ISSUE THREAD

**Issue by LongjunGao** [facebookresearch/vggt-omega#36]
Hello author, thank you very much for your contribution. When reproducing, I found that the point cloud restored by VGGT Omeg does not have the true scale. I hope that the point cloud output by VGGT Omeg can align well with lidar. I plan to use two methods:
The point cloud generated by VGGT Omega was aligned using Sim (3) and then precisely aligned using ICP, but I found that the results were not particularly good. I tested a scenario where its performance was Accuracy (pred → GT RMSE)=1.9561 m and Completeness (GT → pred RMSE)=2.6164 m. Regarding this scale, did vggt Omega consider the issue of scale at the beginning of its design? Or does the author have any relevant experience with good alignment?
2. I want to fine tune VGGT Omega to have the ability to output point clouds with real scales. Does the author have any experience in this area?

**Comment by jytime** [comment]
Hi,

1. The model was trained to recoved the point cloud in a unit space, instead of the metric space.
2. You can check our discussion sections. Basically, you could finetune the model to estimate the metric scale like the style of the camera token of language token. It works well in our early trial.

**Comment by LongjunGao** [comment]
Thanks!
